pkg/cache/tas_flavor_snapshot.go

```diff
@@ -199,17 +201,25 @@ func (s *TASFlavorSnapshot) addCapacity(domainID utiltas.TopologyDomainID, capac
 	s.leaves[domainID].freeCapacity.Add(capacity)
 }
 
+func (s *TASFlavorSnapshot) addNonTASUsage(domainID utiltas.TopologyDomainID, usage resources.Requests) {
```

Comment:
```suggestion
func (s *TASFlavorSnapshot) addNonTASPodsUsage(domainID utiltas.TopologyDomainID, usage resources.Requests) {
```

Here, what exactly does "NonTAS" mean?  I guess that we want to count the Pods with TAS label usage.
I recommend elaborating more like this.

Actually, this input `usage` is came from Pod without TAS label: https://github.com/kubernetes-sigs/kueue/blob/1924d7dc4ef645dc2613149922be7ec39c5573ab/pkg/cache/tas_flavor.go#L103

Reply:
Yes, it comes from Pods without the TAS label, so corresponding to workloads which aren't managed by TAS.

Reply:
> Yes, it comes from Pods without the TAS label, so corresponding to workloads which aren't managed by TAS.

Thanks for explaining that.

> I recommend elaborating more like this.

I drop this offer since we can clarity what is TAS or NonTAS by https://github.com/kubernetes-sigs/kueue/pull/4221#discussion_r1950674814

Reply:
Thanks, while renaming the usage -> tasUsage I also revisted the comment for `freeCapacity` to make it more clear.

Reply:
Great to see! Thanks!